tsfeatures/tsfeatures.py

```diff
@@ -1,37 +1,53 @@
-#!/usr/bin/env python
-# coding: utf-8
+# AUTOGENERATED! DO NOT EDIT! File to edit: ../nbs/tsfeatures.ipynb.
 
+# %% auto 0
+__all__ = ['FREQS', 'acf_features', 'arch_stat', 'count_entropy', 'crossing_points', 'entropy', 'flat_spots', 'frequency',
+           'guerrero', 'heterogeneity', 'holt_parameters', 'hurst', 'hw_parameters', 'intervals', 'lumpiness',
+           'nonlinearity', 'pacf_features', 'series_length', 'sparsity', 'stability', 'stl_features', 'unitroot_kpss',
+           'unitroot_pp', 'tsfeatures']
+
+# %% ../nbs/tsfeatures.ipynb 3
+import os
 import warnings
+from collections import ChainMap
+from functools import partial
+from multiprocessing import Pool
+from typing import Callable, Dict, List, Optional
+
+import pandas as pd
+
+# %% ../nbs/tsfeatures.ipynb 4
 warnings.warn = lambda *a, **kw: False
-import os
+
+# %% ../nbs/tsfeatures.ipynb 5
 os.environ["MKL_NUM_THREADS"] = "1"
 os.environ["NUMEXPR_NUM_THREADS"] = "1"
 os.environ["OMP_NUM_THREADS"] = "1"
-from collections import ChainMap
-from functools import partial
+
+# %% ../nbs/tsfeatures.ipynb 6
 from itertools import groupby
-from math import log, e
-from multiprocessing import cpu_count, Pool
-from typing import List, Dict, Optional, Callable
+from math import e  # maybe change with numpy e
+from typing import Dict
 
 import numpy as np
 import pandas as pd
 from antropy import spectral_entropy
 from arch import arch_model
 from scipy.optimize import minimize_scalar
 from sklearn.linear_model import LinearRegression
-from statsmodels.api import add_constant, OLS
+from statsmodels.api import OLS, add_constant
 from statsmodels.tsa.ar_model import AR
 from statsmodels.tsa.holtwinters import ExponentialSmoothing
 from statsmodels.tsa.seasonal import STL
-from statsmodels.tsa.stattools import acf, pacf, kpss
+from statsmodels.tsa.stattools import acf, kpss, pacf
 from supersmoother import SuperSmoother
 
-from .utils import (embed, FREQS, hurst_exponent,
-                    lambda_coef_var, poly,
-                    scalets, terasvirta_test, ur_pp)
+from .utils import *
 
+# %% ../nbs/tsfeatures.ipynb 7
+FREQS = {"H": 24, "D": 1, "M": 12, "Q": 4, "W": 1, "Y": 1}
```

Comment:
This is defined in utils as well, please remove it from here

Reply:
removed